test.js

```diff
@@ -1,6 +1,26 @@
 'use strict';
 var test = require('ava');
 var osLocale = require('./');
+var envVars = ['LC_ALL', 'LANGUAGE', 'LANG', 'LC_MESSAGES'];
+var expectedFallback = 'en_US';
+
+function reloadOsLocale() {
+	delete require.cache[require.resolve('./')];
```

Comment:
You could use https://github.com/sindresorhus/clear-require or https://github.com/sindresorhus/require-uncached here.